scripts/windows/setup.template.iss

```diff
@@ -12,7 +12,7 @@ AppName={{AppName}}
 AppVersion={{Version}}
 DefaultDirName={pf}\{{AppName}}
 DefaultGroupName={{AppName}}
-UninstallDisplayIcon={app}\{{AppExecutableName}}
+UninstallDisplayIcon={{AppIcon}}
```

Comment:
I think due to the same issue, this was pointing at `Oni2.exe` which has no icon, so the uninstaller would have no icon.

Reply:
Thanks for the details, @CrossR !